src/vorta/tray_menu.py

```diff
@@ -71,5 +71,6 @@ def set_tray_icon(self, active=False):
         Use white tray icon, when on Gnome or in dark mode. Otherwise use dark icon.
         """
         icon_name = f"icons/hdd-o{'-active' if active else ''}.png"
+        self.setToolTip("Vorta\n Status: Running" if active else "Vorta\n Status: Idle")
```

Comment:
```suggestion
        self.setToolTip(self.tr("Vorta\nStatus: Running") if active else self.tr("Vorta\nStatus: Idle"))
```

Should mark any user-readable text for translation.

Reply:
Thanks @samuel-w
I added now "Next Backup ..."  and it works, but the flake8 check report:
./src/vorta/tray_menu.py:74:121: E501 line too long (151 > 120 characters)
!

Reply:
flake8 resolved.
What is about last successful Task/Backup? How to find this def?

Reply:
Use this command 
`EventLogModel.select().where(EventLogModel.subcommand == 'create').select(pw.fn.MAX(EventLogModel.start_time)).scalar()`

Reply:
Thank you @samuel-w 
this command will list last started backup, but not last successful backup. Unfortunately there is no entry whether a task is successful or not. The "returncode" in DB is not to be trusted.
i did workaround with this and it works:

`last_backup = EventLogModel.select()\
            .where(EventLogModel.subcommand == 'create')\
            .where(EventLogModel.returncode == '1')\
            .where(EventLogModel.repo_url != '(NULL)').select(pw.fn.MAX(EventLogModel.start_time)).scalar()
        last_backup_formatted = last_backup.strftime('%d %B %H:%M')
        icon_name = f"icons/hdd-o{'-active' if active else ''}.png"
        self.setToolTip(self.tr("Vorta\nStatus: Running") if active else
                        self.tr("Vorta\nStatus: Idle\nLast Backup: %s")
                        % last_backup_formatted + self.tr("\nNext Backup: %s" % self.app.scheduler.next_job))`

But i am not sure if this the best practice!